scripts/VirtualizationTools.ps1

```diff
@@ -1,31 +1,3 @@
-
-choco install -y Microsoft-Windows-Subsystem-Linux -source windowsfeatures
 choco install -y Microsoft-Hyper-V-All -source windowsFeatures
-# alternative to above: Enable-WindowsOptionalFeature -Online -NoRestart -FeatureName Microsoft-Windows-Subsystem-Linux
-# alternative to above: Enable-WindowsOptionalFeature -Online -NoRestart -FeatureName Microsoft-Hyper-V 
 choco install -y docker-for-windows

```

Comment:
I would have one for docker, not virtualization.

Reply:
my statement is it feels overreaching from the title.  I could see someone misunderstanding this.

Also what if someone is on not a SKU of windows that doesn't support Hyper-V (aka home edition), what happens?

Reply:
separate docker makes sense.  and then name this one Hyper-V

Reply:
When a helper script is down to 1-line we should consider getting rid of it

Reply:
I'll make some changes to this PR and we may need to revisit again later